js/components/dataset/resource/file-form.vue

```diff
@@ -57,10 +57,12 @@
 import API from 'api';
 import Dataset from 'models/dataset';
 import CommunityResource from 'models/communityresource';
+import FormHorizontal from 'components/form/horizontal-form.vue';
```

Comment:
Yep, trying to make some minimal changes (even with ES6 changes).
I added this case to #753